modules/nanomethphase/1.0/config/default.yaml

```diff
@@ -0,0 +1,39 @@
+lcr-modules:
+    
+    nanomethphase:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            sample_bai: "__UPDATE__"
+            meth_calls: "__UPDATE__"
+            vcf: "__UPDATE__" #needs to be a vcf with gt info and variants from a single sample in column 10 
+			      #ie if run matched tumour normal vcf should contain only calls from tumour, 
+			      #or leave blank to use clair3 variant calls
+        
+        scratch_subdirectories: []
+
+        conda_envs:
+            nanomethphase: "{MODSDIR}/envs/nanomethphase.yaml"
+            clair3: "{MODSDIR}/envs/clair3.yaml"
+            whatshap: "{MODSDIR}/envs/whatshap.yaml"
+
+        threads:
+            methyl_call: 20
+            nanomethphase: 50
+            clair3: 50
+            
+        mem_mb:
+            methyl_call: 25000 
+            clair3: 25000
+            whatshap: 25000
+            
+        clair3:
+            model: "{CONDA_PREFIX}/bin/models/{model_name}" #Update to conda prefix of Clair3 environment and select appropriate model recommended r941_prom_hac_g360+g422 for GSC bams
```

Comment:
This can be pretty hard to track down because of the way Snakemake names Conda directories. 

In the vcf2maf module we have a function that identifies the path to the GAMBLR conda environment: 

```
import hashlib
# Obtain the path to the GAMBLR conda environment
md5hash = hashlib.md5()
if workflow.conda_prefix: 
    conda_prefix = workflow.conda_prefix
else: 
    conda_prefix = os.path.abspath(".snakemake/conda")

md5hash.update(conda_prefix.encode())
f = open("config/envs/GAMBLR.yaml", 'rb')
md5hash.update(f.read())
f.close()
h = md5hash.hexdigest()
GAMBLR = glob.glob(conda_prefix + "/" + h[:8] + "*")[0]
```

You could use this to automagically retrieve the path to the Conda env, then the config value would only have to be the model name.

Reply:
Clair3 has been moved to phase_vcf module to avoid reproducing vcf files used by this module and whatshap module